arcaflow_plugin_pcp/pcp_schema.py

```diff
@@ -1,16 +1,19 @@
 import typing
+import re
 from dataclasses import dataclass
-from arcaflow_plugin_sdk import plugin, schema
+from arcaflow_plugin_sdk import plugin, schema, validation
 
 validation_warning = (
     " NOTE: Input not validated by the plugin --"
     " Any errors are likely to be produced at the end of the plugin run and"
     " may result in workflow failures."
 )
 
+file_path_pattern = re.compile(r"((?:[^\/]*\/)*)(.*)")
```

Comment:
I'm not exactly sure what this pattern is supposed to *reject* ... it looks rather like it'll accept pretty much anything with or without slashes. 😆

Reply:
Yeah, I didn't like it much either, and I'm not sure how reasonable it is to even write a pattern to validate a posix file path since almost anything is acceptable.

Reply:
Let's see...a file path consists of zero or more directories, each followed by a forward-slash, followed by a filename.  Pathname components contain at least one character but not a forward-slash (and anyone who wants to argue that that should be permitted can...file a request for enhancement...), but sequences of slashes are acceptable.  So, the pattern should look something like this:
```python
file_path_pattern = re.compile(r"((?:[^/]*/)*)([^/]+)")
```
Note that you need to anchor the match appropriately, to ensure that it is applied to the entire path (otherwise, it will, e.g., match the last directory as the file and ignore the trailing slash and anything following it).

(BTW, I'm not sure why your expression was trying to escape the forward-slashes.)